core/src/test/scala/com/salesforce/op/stages/impl/feature/DateMapVectorizerTest.scala

```diff
@@ -122,7 +122,7 @@ class DateMapVectorizerTest extends FlatSpec with TestSparkContext with Attribut
   }
 
   it should "vectorize dates correctly on test date" in {
-    checkAt(new JDateTime(2017, 9, 28, 15, 45, 39, DateTimeUtils.DefaultTimeZone))
+    checkAt(new JDateTime(1901, 1, 1, 0, 0, 0, DateTimeUtils.DefaultTimeZone))
```

Comment:
better add a new test case

Reply:
@TuanNguyen27 please add test cases for ArithmeticExceptions you  are trying to solve

Reply:
@gerashegalov the new test case is `checkAt(new JDateTime(1901, 1, 1, 0, 0, 0, DateTimeUtils.DefaultTimeZone))`, which exceed the limit if we use `Days.daysBetween`. Does that work ?